scripts/coverage_report.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Check that the test directory and report path arguments are provided
+if [ $# -ne 2 ]
+then
+	echo "Usage: coverage_report.sh path/to/tests path/to/report/destination"
+	echo "/path/to/tests should be '.' for coverage to be computed for all modules"
+	echo "To get coverage for a subset of modules set /path/to/tests to the appropriate subdirectory"
+	exit 1
+fi
+
+# Set the directory containing the tests to run (includes subdirectories)
+TEST_DIR=$1
+
+# Set the directory to which the report will be saved
+COVERAGE_DIR=$2
+
+# This needs to run in libra
```

Comment:
This isn't quite what this is checking, correct?  You seem to need a `cd "$(dirname $DIR)"` on the `else` branch of the `if` below. 

To convince yourself of that, you can consider line 70 and its dependence on `target` being where you think you are. Or try to run from `libra/scripts`